tests/integration/test_models/standard_output_tests.py

```diff
@@ -328,7 +328,14 @@ def test_conservation(self):
             + self.n_SEI_p_av(self.solution.t) * L_p * A
         )
         diff = (self.c_s_tot[1:] - self.c_s_tot[:-1]) / self.c_s_tot[:-1]
-        np.testing.assert_array_almost_equal(diff, 0)
+        if "profile" in self.model.options["particle"]:
+            np.testing.assert_array_almost_equal(diff, 0, decimal=10)
+        elif self.model.options["surface form"] == "differential":
+            np.testing.assert_array_almost_equal(diff, 0, decimal=10)
+        elif self.model.options["SEI"] == "ec reaction limited":
```

Comment:
I am not 100% that the `ec reaction limited` is conservative. In any case, with this new tolerance it should be fine.

Reply:
need to think a little bit more carefully about factors of 2 in the SEI layer growth equation